bazel/candid.bzl

```diff
@@ -14,16 +14,18 @@ if [[ $mr_title == *"[override-didc-check]"* ]]; then
     exit 0
 fi
 
-# https://docs.gitlab.com/ee/ci/variables/predefined_variables.html
-# The HEAD SHA of the target branch of the merge request. The variable is empty in merge request pipelines. 
-# The SHA is present only in merged results pipelines.
-readonly merge_base=${{CI_MERGE_REQUEST_TARGET_BRANCH_SHA:-HEAD}}
+if [ -z "${{CI_MERGE_REQUEST_TARGET_BRANCH_NAME:-}}" ]; then
```

Comment:
I think it would be a little cleaner to just add this to the workflow:

```
CI_MERGE_REQUEST_TARGET_BRANCH_SHA: ${{ github.event.pull_request.base.sha }}
```

And then you can keep the existing logic.

Reply:
Thanks for the suggestion! I was searching for an environment variable but could only find one for the name of the base branch but not for the SHA. Let me see if this works...